test/models/story_test.rb

```diff
@@ -171,6 +171,14 @@
         story.unpublish!
       end.must_raise(RuntimeError)
     end
+
+    it 'allows the published date to be set via boolean' do
+      story.published = false
+      story.published.must_be_nil
```

Comment:
didn't you remove the `event_attribute :published_at` in this same PR? (I assumed this spec was to test the `published=` method, which takes care of setting `published_at` -- so wouldn't you want to check the `published at` attribute? Or am I confusing myself...)